lib/livebook.ex

```diff
@@ -3,13 +3,76 @@ defmodule Livebook do
   Livebook is an interactive notebook system for Elixir.
 
   This module includes the public API.
+
+  ## Configuration
+
+  See `config_runtime/0` for bootstrapping the default runtime
+  configuration. There are several public configuration entries that
+  you can customize.
+
+  ### Custom plugs
+
+  You can list a number of plugs to call directly before the Livebook
+  router
+
+      config :livebook, :plugs, [{CustomPlug, []}]
+
+  ### Embedded runtime dependencies
+
+  In case you use the Embedded runtime and support installing dependencies
+  with `Mix.install/2`, you can make those discoverable in the
+  dependency search, by configuring entries like this:
+
+      config :livebook, Livebook.Runtime.Embedded,
+        dependency_entries: [
```

Comment:
Instead of specifying a list, I think we should rather support a module that we call a function on, and that can return a list?